### PR TITLE
Adds PostgreSQL scheme. Adds simple CRUD operations

### PR DESCRIPTION
**Pull request by k1rill-fedoseev**
I have added a proposed db schema as an sql dump file at `backend/app/app/db/init.sql`.
Correspondent docker-compose `db` service was also updated.

Simple CRUD operations (creating students/trainers, creating/deleting sport groups, inserting quiz answers, etc.) were also added.

### CODE REVIEW

File: backend/app/app/db/connection.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import os
+import psycopg2
+
+POSTGRES_USER = os.environ['POSTGRES_USER']
+POSTGRES_PASSWORD = os.environ['POSTGRES_PASSWORD']
+POSTGRES_DB = os.environ['POSTGRES_DB']
+
+
+def create_connection():
```
Comment (WinnerOK) [inline backend/app/app/db/connection.py]: Connection credentials are present in `core.config`

File: backend/app/app/main.py [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
-
-@app.middleware("http")
-async def db_session_middleware(request: Request, call_next ):
-    logger.debug("Entered middleware")
-    request.state.db = Session()
-    response = await call_next(request)
-    request.state.db.close()
-    return response
-
+# from fastapi import FastAPI
```
Comment (WinnerOK) [inline backend/app/app/main.py]: Why did you comment all server code here?

File: backend/app/app/db/connection.py (hunk 1 above)
Comment (k1rill-fedoseev) [reply backend/app/app/db/connection.py]: Thanks. Updated the code with usage of `core.config` instead.

File: backend/app/app/main.py (hunk 2 above)
Comment (k1rill-fedoseev) [reply backend/app/app/main.py]: Ok, uncommented it.

**Comment by gapurinov** [comment]
Can you rewrite CRUD for groups using pedantic model?

**Pull request merged**